Sources/Apodini/Semantic Model Builder/gRPC/ProtobufferBuilder/Extensions/Model+EnrichedInfo.swift

```diff
@@ -35,20 +54,40 @@ extension ProtobufferMessage.Property {
     }
 }
 
-extension ProtobufferMessage {
-    init?(_ node: Node<Property?>) {
-        // If a child is nil, there is a circle in theory.
-        // Thus, this message is incomplete.
-        // However, a complete message was built closer to the root of the tree.
-        let properties = node.children.compactMap { $0.value }
-        guard properties.count == node.children.count,
-              let name = node.value?.typeName else {
-            return nil
+fileprivate extension TypeInfo {
+    func compatibleName() throws -> String {
+        if isSupportedScalarType(type) {
+            let infix = isAmbiguousSupportedFixedWidthInteger(type)
+                ? String(describing: Int.bitWidth)
+                : ""
+            let suffix = isSupportedScalarType(type)
+                ? ""
+                : "Message"
```

Comment:
I would suggest adding if statements here to make the code a bit easier to read.